pkgs/development/python-modules/markdown-it-py/default.nix

```diff
@@ -1,41 +1,39 @@
 { lib, buildPythonPackage, fetchFromGitHub, pytestCheckHook, pythonOlder
 , attrs
 , coverage
+, linkify-it-py
 , psutil
 , pytest-benchmark
+, pytest-cov
+, pytest-regressions
+, typing-extensions
 }:
 
 buildPythonPackage rec {
   pname = "markdown-it-py";
-  version = "0.6.2";
+  version = "1.0.0";
 
   disabled = pythonOlder "3.7";
```

Comment:
Python > 3.6, see [here](https://github.com/executablebooks/markdown-it-py/blob/master/setup.cfg#L34).